### DISCUSSION TITLE
Nvchad terminal not working

### DISCUSSION THREAD

**Discussion by BSIT-Evanism** [NvChad/NvChad#1800] [Q&A]
Im currently using nvchad, i had all the pre requisite installed, Im currently using hyper as my terminal and I use it to open nvim, whenever I open the NV term, its says bash.exe no file is found etc![rn_image_picker_lib_temp_2547c803-16d4-46b2-9d0a-0ec861af430a.jpg](https://user-images.githubusercontent.com/71175442/223144729-3c318285-1a2c-47df-9b30-1603f2ac1c5d.jpg)

**Comment by siduck** [comment]
@BSIT-Evanism are you  on windows?

**Reply by BSIT-Evanism** [reply]
Yes
